dashboard/src/containers/RoutesContainer/Routes.tsx

```diff
@@ -26,8 +26,8 @@ const privateRoutes: {
   "/apps/ns/:namespace/:releaseName": AppViewContainer,
   "/apps/ns/:namespace/new/:repo/:id/versions/:version": AppNewContainer,
   "/apps/ns/:namespace/upgrade/:releaseName": AppUpgradeContainer,
-  "/charts": ChartListContainer,
-  "/charts/:repo": ChartListContainer,
+  "/catalog": CatalogListContainer,
+  "/catalog/:repo": CatalogListContainer,
```

Comment:
For now, I'm not changing the ChartView routes, but this looks a little bit confusing. One option is to not change this to `/catalog/:repo` and keep it as `/charts/:repo`, but keep `/catalog`, but then the container is called CatalogListContainer. I'm not sure what the best option here is.